node/node.go

```diff
@@ -117,17 +117,17 @@ func DefaultNewNode(config *cfg.Config, logger log.Logger) (*Node, error) {
 }
 
 // MetricsProvider returns a consensus, p2p and mempool Metrics.
```

Comment:
[no change needed] this godoc appears inaccurate b/c the function has 4 return parameters but I expect this is incorrect in upstream.

```go
// MetricsProvider returns a consensus, p2p, mempool, and state metrics.
```

Reply:
Yeah it's just the same